docs/components/table.md

```diff
@@ -99,6 +99,10 @@ Set this to `true` to limit pagination navigation to `previous` / `next` page on
 
 Set this to `true` to remove the pagination bar when using a fetcher.
 
+### hidePaginationWhenOptional
+
+Set this to `true` to remove the pagination bar. For cases, when the table record in Konnect is less than or equal to 15.
```

Comment:
Rewording since we don't want to mention Konnect in the docs.

Also removing `15` as the logic appears to hide the pagination if less than the page count (@aanchalm01 assuming this is correct?)

```suggestion
Set this to `true` to hide the pagination UI when the table record count is less than or equal to the page size.
```

Reply:
I am confirming about the record number.

Reply:
Done!